Source/WebKit/UIProcess/ios/forms/WKDatePickerPopoverController.mm

```diff
@@ -105,6 +117,15 @@ - (UIToolbar *)accessoryView
     return _accessoryView.get();
 }
 
+- (CGSize)estimatedMaximumPopoverSize
+{
+    constexpr auto additionalHeightToAvoidClippingToolbar = 60;
```

Comment:
Can you explain the origin of this value?

Reply:
> Can you explain the origin of this value?

Unfortunately, this was just determined by manual testing on an iPad and iPhone. I believe it's some additional padding related to the popover UI itself (e.g. height of the arrow, and margins around the content view).